websubhub-examples/kafka-hub/consolidator/.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
Shall we move the .gitignore file into the project root (`websubhub-example` directory)?